src/fmopl.c

```diff
@@ -501,7 +501,7 @@ INLINE void OPL_CALC_CH( OPL_CH *CH )
 	}
 }
 
-/* ---------- calcrate rythm block ---------- */
+/* ---------- calcrate rhythm block ---------- */
```

Comment:
Should this be "calculate"?

Reply:
@Malvineous `calcrate`appears in other parts of the code...so I wasn't sure if it was referencing this internal jargon

Reply:
@Malvineous can you confirm?

Reply:
It's hard to say but I can't think what other word it could be.  I think change them all to "calculate", unless you can find any that have nothing to do with mathematical operations.